src/ExportMap.js

```diff
@@ -525,31 +526,34 @@ ExportMap.parse = function (path, content, context) {
     });
     try {
       if (tsConfigInfo.tsConfigPath !== undefined) {
-        const jsonText = fs.readFileSync(tsConfigInfo.tsConfigPath).toString();
-        if (!parseConfigFileTextToJson) {
-          // this is because projects not using TypeScript won't have typescript installed
-          ({ parseConfigFileTextToJson } = require('typescript'));
-        }
-        return parseConfigFileTextToJson(tsConfigInfo.tsConfigPath, jsonText).config;
+        // Projects not using TypeScript won't have `typescript` installed.
+        if (!ts) (ts = require('typescript'));
+  
+        const configFile = ts.readConfigFile(tsConfigInfo.tsConfigPath, ts.sys.readFile);
+        return ts.parseJsonConfigFileContent(
```

Comment:
can you confirm which versions of typescript offer this method?

Reply:
I'm looking now. I see references to this from 2015:
https://github.com/microsoft/TypeScript/issues/5276

Based on the dates here, that would indicate it was around pre-v3.
https://github.com/microsoft/TypeScript/releases?after=v3.0.0

Reply:
It looks like it was added around 1.7.2.
https://github.com/Microsoft/TypeScript/blob/v1.7.2/src/compiler/commandLineParser.ts#L416

It's not in 1.6.2, at least not in this file.

Reply:
We only test on v2 and v3 (and should add v4) so I'm comfortable not caring about old v1. Thanks!